lib/spack/spack/package.py

```diff
@@ -1669,10 +1686,11 @@ def build_process():
                 # Ensure the metadata path exists as well
                 mkdirp(spack.store.layout.metadata_path(self.spec), mode=perms)
 
-            # Fork a child to do the actual installation
-            # we preserve verbosity settings across installs.
-            PackageBase._verbose = spack.build_environment.fork(
-                self, build_process, dirty=dirty, fake=fake)
+            # Fork a child to do the actual code installation.
+            # Preserve verbosity settings across installs.
+            if self.has_code:
+                PackageBase._verbose = spack.build_environment.fork(
```

Comment:
Removed the check

Reply:
I forgot that I used the check to avoid the problem of not having a `spack-src` directory associated with no-source (or bundle) packages.  (Refer to the failing tests from August 12, 2019.)